buck/cmd.py

```diff
@@ -1,9 +1,9 @@
 import argparse
-import configparser
 import copy
 import os
 import subprocess
 
+import utils
```

Comment:
the snap fails importing utils this way, it should be better to use "from buck import utils"